project/__main__.py

```diff
@@ -0,0 +1,801 @@
+'''
+(The features of this codejam project are not to be taken seriously)
+Code Jam 4 submission for High Houses of Adorable Laboratories: HTT
+HTT is used to run a text editor with a built-in onscreen keyboard.
+Go ahead, give it a try! Our tutorial will walk you through how to use it!
+'''
+import asyncio
+import os
+import threading
+import itertools
+import tkinter as tk
+import tkinter.font as tkFont
+from tkinter import filedialog as fd
+from tkinter import messagebox
+from tkinter import ttk
+import json
+import random
+from pathlib import Path
+import wave
+from PIL import Image, ImageTk
+import pyaudio
+
+
+SCRIPT_DIR = Path(__file__).parent
+
+KEY_DESCRIPTION_PATH = SCRIPT_DIR / Path('key_descriptions.json')
```

Comment:
great use of the `/` operator overload in pathlib, however this actually works with just a string on the right side of the operator.

```py
# This works fine
SCRIPT_DIR / "key_descriptions.json"
```

Reply:
I did not know that, thank you!